scripts/cli/tasks/closeMilestone.ts

```diff
@@ -0,0 +1,87 @@
+import axios from 'axios';
+import { Task, TaskRunner } from './task';
+
+interface CloseMilestoneOptions {
+  milestone: string;
+}
+
+const closeMilestoneTaskRunner: TaskRunner<CloseMilestoneOptions> = async ({ milestone }) => {
+  const gitHubUsername = process.env.GITHUB_USERNAME;
+  const gitHubToken = process.env.GITHUB_ACCESS_TOKEN;
+
+  if (!gitHubUsername || !gitHubToken) {
+    console.log('the close milestone operation needs a GITHUB_USERNAME and GITHUB_ACCESS_TOKEN environment variables');
+    return;
+  }
+
+  const client = axios.create({
+    baseURL: 'https://api.github.com/repos/grafana/grafana',
+    timeout: 10000,
+    auth: {
+      username: gitHubUsername,
+      password: gitHubToken,
+    },
+  });
+
+  const cherryPickLabel = 'cherry-pick needed';
```

Comment:
Would it be better to move this closer to where is used? I was re-reading the file (now that I've some context of what the PR is trying to achieve) and had to stop on L49 to figure out were that was coming from.